src/openrct2/ride/Track.h

```diff
@@ -582,7 +582,6 @@ uint8 track_element_get_ride_index(const rct_tile_element * tileElement);
 void track_element_set_ride_index(rct_tile_element * tileElement, uint8 rideIndex);
 uint8 track_element_get_type(const rct_tile_element * tileElement);
 void track_element_set_type(rct_tile_element * tileElement, uint8 rideIndex);
-
```

Comment:
Could you revert this bit? (Yes, I'm nitpicky.)

Reply:
Yeah, must've gotten in when I was experimenting with creating a function for the track block array.